weather.py

```diff
@@ -0,0 +1,212 @@
+import argparse
+import os
+import calendar
+
+months = list(calendar.month_abbr)[1:]
+
+labels = [
+    'PKT', 'Max TemperatureC', 'Mean TemperatureC',
+    'Min TemperatureC', 'Dew PointC', 'MeanDew PointC',
+    'Min DewpointC', 'Max Humidity', 'Mean Humidity',
+    'Min Humidity', 'Max Sea Level PressurehPa',
+    'Mean Sea Level PressurehPa', 'Min Sea Level PressurehPa',
+    'Max VisibilityKm', 'Mean VisibilityKm', 'Min VisibilitykM',
+    'Max Wind SpeedKm/h', 'Mean Wind SpeedKm/h', 'Max Gust SpeedKm/h',
+    'Precipitationmm', 'CloudCover', 'Events', 'WindDirDegrees'
+]
+
+month_abbr = dict(zip(range(1, 13), calendar.month_abbr[1:]))
+
+
+def load_weather_data(file_path, year):

```

Comment:
This function is in violation of SRP Principle. We need to ensure that our all functions are in complete compliance with SRP.

Reply:
Name of parameters also requires a reconsideration.